src/lib/inc/util.h

```diff
@@ -9,81 +9,78 @@
 /* UTILITY MACROS */
 
 /* align VAL to TO which must be power a two */
-#define ALIGN(VAL, TO) ((((VAL) + (TO)-1) / (TO)) * TO)
-#define IS_ALIGNED(VAL, TO) (!((VAL)%(TO)))
+#define ALIGN(VAL, TO)       ((((VAL) + (TO)-1) / (TO)) * TO)
+#define IS_ALIGNED(VAL, TO)  (!((VAL) % (TO)))
 #define ALIGN_FLOOR(VAL, TO) ((VAL) & ~((TO)-1))
-#define NUM_PAGES(SZ) (ALIGN(SZ, PAGE_SIZE)/PAGE_SIZE)
-#define PAGE_OFFSET_MASK ((PAGE_SIZE)-1)
-#define PAGE_FRAME_MASK (~(PAGE_OFFSET_MASK))
+#define NUM_PAGES(SZ)        (ALIGN(SZ, PAGE_SIZE) / PAGE_SIZE)
+#define PAGE_OFFSET_MASK     ((PAGE_SIZE)-1)
+#define PAGE_FRAME_MASK      (~(PAGE_OFFSET_MASK))
 
-#define SR_OR(VAL, SHIFT) (((VAL) >> (SHIFT)) | VAL)
+#define SR_OR(VAL, SHIFT)    (((VAL) >> (SHIFT)) | VAL)
 /* Next Power Of Two */
-#define NPOT(VAL)                                                     \
-    ((SR_OR(((VAL)-1), 1) | SR_OR(SR_OR(((VAL)-1), 1), 2) |           \
-      SR_OR(SR_OR(SR_OR(((VAL)-1), 1), 2), 4) |                       \
-      SR_OR(SR_OR(SR_OR(SR_OR(((VAL)-1), 1), 2), 4), 8) |             \
-      SR_OR(SR_OR(SR_OR(SR_OR(SR_OR(((VAL)-1), 1), 2), 4), 8), 16)) + \
-     1)
+#define NPOT(VAL)                                                        \
+    ((SR_OR(((VAL)-1), 1) | SR_OR(SR_OR(((VAL)-1), 1), 2) |              \
+         SR_OR(SR_OR(SR_OR(((VAL)-1), 1), 2), 4) |                       \
+         SR_OR(SR_OR(SR_OR(SR_OR(((VAL)-1), 1), 2), 4), 8) |             \
+         SR_OR(SR_OR(SR_OR(SR_OR(SR_OR(((VAL)-1), 1), 2), 4), 8), 16)) + \
+        1)
 
 /* Previous Power Of Two */
-#define PPOT(VAL) (NPOT((VAL)) - (NPOT((VAL)) >> 1))
+#define PPOT(VAL) (NPOT(VAL) - (NPOT(VAL) >> 1))
 
-#define STR(s)  #s
-#define XSTR(s)  STR(s)
+#define STR(s)    #s
+#define XSTR(s)   STR(s)
 
 #ifndef __ASSEMBLER__
 
 #define DEFINE_OFFSET(SYMBOL, STRUCT, FIELD) \
     asm volatile("\n-> " XSTR(SYMBOL) " %0 \n" : : "i"(offsetof(STRUCT, FIELD)))
 
-#define DEFINE_SIZE(SYMBOL, TYPE) \
-    asm volatile("\n-> " XSTR(SYMBOL) " %0 \n" : : "i"(sizeof(TYPE)))
+#define DEFINE_SIZE(SYMBOL, TYPE) asm volatile("\n-> " XSTR(SYMBOL) " %0 \n" : : "i"(sizeof(TYPE)))
```

Comment:
both macros are being wrapped differently.

Reply:
This happens because the macro definition now fits the 100 chars.